nablapps/exchange/views.py

```diff
@@ -6,23 +6,26 @@
 
 from .models import RETNINGER, Exchange, ExchangeNewsArticle, Info, University
 
-# temporary
-
 
 class ExchangeFrontpageView(ListView):
+    model = University
     template_name = "exchange/exchange_frontpage.html"
-    model = ExchangeNewsArticle
-    context_object_name = "news_list"
-    paginate_by = 5
-    queryset = ExchangeNewsArticle.objects.order_by("-pk")
+    context_object_name = "ex_list"
+
+    def get_queryset(self):
+        query = self.request.GET.get("q")
+        queryset = University.objects.order_by("land")
+        if query:
+            queryset = queryset.filter(
+                Q(land__icontains=query) | Q(univ_navn__icontains=query)
+            )
```

Comment:
v cool

Reply:
I cannot take credit for those lines however, they were part of the old view for the university list which was not authored by me. I only slightly restructured the views and moved the lines around :upside_down_face: